### ISSUE TITLE
bode_plot phase wrapping incorrect for multiple systems.

### ISSUE THREAD

**Issue by henklaak** [python-control/python-control#862]
Minimum example:
```
# two RHP's
G = zpk([],[1,1], gain=1)  
bode_plot(G)
bode_plot((G,G))
```

The second call shows a figure with different phases for each (identical) system.

![bode1](https://user-images.githubusercontent.com/3962727/219655726-fae608b9-6cbc-4d16-a1f3-69d846910276.png)
![bode2](https://user-images.githubusercontent.com/3962727/219655742-e6b12471-61fd-4fed-92a3-e9fef5ee25a4.png)

**Comment by henklaak** [comment]
I'm working a PR for it.

**Comment by murrayrm** [comment]
Fixed in #863
